src/System.Net.Http/ref/4.0.0/System.Net.Http.cs

```diff
@@ -0,0 +1,607 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+
+namespace System.Net.Http
```

Comment:
This entire file should be deleted.  Please use a depproj to redist the previously shipped reference assembly for 4.0.0.0